internal/fsutil/file_system_cleaner_test.go

```diff
@@ -0,0 +1,74 @@
+package fsutil_test
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/wal-g/wal-g/internal/fsutil"
+)
+
+func TestGetFiles_DirectoryNotExist(t *testing.T) {
+	var cleaner = fsutil.FileSystemCleaner{}
+
+	_, err := cleaner.GetFiles("test_data/NotExist")
+
+	assert.Error(t, err)
+}
+
+func TestGetFiles_OneFile(t *testing.T) {
+	var cleaner = fsutil.FileSystemCleaner{}
+	expected := []string{"file.txt"}
+
+	result, err := cleaner.GetFiles("test_data/directory_with_one_file")
```

Comment:
Lets create and delete the mock files dynamically like it is done in https://github.com/wal-g/wal-g/blob/d67de444c669e474eaec568a590a9441d83f4e73/internal/configure_test.go#L131